documents.go

```diff
@@ -133,3 +133,75 @@ func (cl *DocumentsCreateClient) Do(ctx context.Context) (*Document, error) {
 
 	return success.Data, nil
 }
+
+// documentsUpdateParams represents the Outline Documents.update parameters
+type documentsUpdateParams struct {
+	Id      DocumentID `json:"id"`
+	Title   string     `json:"title,omitempty"`
+	Text    string     `json:"text,omitempty"`
+	Append  bool       `json:"append,omitempty"`
+	Publish bool       `json:"publish,omitempty"`
+	Done    bool       `json:"done,omitempty"`
+}
+
+// DocumentsUpdateClient is a client for updating a single document.
+type DocumentsUpdateClient struct {
+	sl     *sling.Sling
+	params documentsUpdateParams
+}
+
+func newDocumentsUpdateClient(sl *sling.Sling, params documentsUpdateParams) *DocumentsUpdateClient {
+	copy := sl.New()
+
+	return &DocumentsUpdateClient{sl: copy, params: params}
+}
+
+// Update returns a client for updating a single document in the specified collection.
+// API reference: https://www.getoutline.com/developers#tag/Documents/paths/~1documents.update/post
+func (cl *DocumentsClient) Update(id DocumentID) *DocumentsUpdateClient {
+	return newDocumentsUpdateClient(cl.sl, documentsUpdateParams{Id: id})
+}
+
+func (cl *DocumentsUpdateClient) Title(title string) *DocumentsUpdateClient {
+	cl.params.Title = title
+	return cl
+}
+
+func (cl *DocumentsUpdateClient) Text(text string) *DocumentsUpdateClient {
+	cl.params.Text = text
+	return cl
+}
+
+func (cl *DocumentsUpdateClient) Publish(publish bool) *DocumentsUpdateClient {
+	cl.params.Publish = publish
+	return cl
+}
+
+func (cl *DocumentsUpdateClient) Append(append bool) *DocumentsUpdateClient {
+	cl.params.Append = append
+	return cl
+}
+
+func (cl *DocumentsUpdateClient) Done(done bool) *DocumentsUpdateClient {
```

Comment:
This parameter is marked as
> This property will soon be deprecated.

Not sure if we still want to support it or not 🤔

Reply:
Yeah, was also not sure if we want to support that - but I think we should support what they support and remove what they remove (?) If that makes sense. I'd just go with whatever they currently provide and remove it when they do

Reply:
Yep, fine by me 🙃